cmd/utils.go

```diff
@@ -1261,3 +1261,14 @@ func MockOpenIDTestUserInteraction(ctx context.Context, pro OpenIDClientAppParam
 	// fmt.Printf("TOKEN: %s\n", rawIDToken)
 	return rawIDToken, nil
 }
+
+// unwrapAll will unwrap the returned error completely.
+func unwrapAll(err error) error {
+	for {
```

Comment:
Can we add one Unwrap outside to avoid the loop?

Reply:
Why? @harshavardhana 

The point is that it *is* a loop, since errors can be wrapped multiple times.

Reply:
[errors.Unwrap](https://pkg.go.dev/errors#Unwrap) is not recursive, only returns the next error, that may be wrapped.

Reply:
okay @klauspost